src/build/rustc.rs

```diff
@@ -25,24 +25,29 @@ use self::rustc_save_analysis::CallbackHandler;
 use self::syntax::ast;
 use self::syntax::codemap::{FileLoader, RealFileLoader};
 
-use build::{BufWriter, BuildResult};
+use config::Config;
+use build::{BufWriter, BuildResult, Environment};
 use data::Analysis;
 use vfs::Vfs;
 
 use std::collections::HashMap;
-use std::env;
 use std::ffi::OsString;
 use std::io;
 use std::path::{Path, PathBuf};
 use std::sync::{Arc, Mutex};
 
 // Runs a single instance of rustc. Runs in-process.
-pub fn rustc(vfs: &Vfs, args: &[String], envs: &HashMap<String, Option<OsString>>, build_dir: &Path) -> BuildResult {
+pub fn rustc(vfs: &Vfs, args: &[String], envs: &HashMap<String, Option<OsString>>, build_dir: &Path, rls_config: Arc<Mutex<Config>>) -> BuildResult {
```

Comment:
I believe you forgot to add the argument at the `rustc` [call site](https://github.com/rust-lang-nursery/rls/pull/414/files#diff-9997203f2de5b62d7810f98eebd0cb72R396)